app/views/dashboards/_project.html.erb

```diff
@@ -0,0 +1,8 @@
+<tr class="<%= 'warning' if deploy_group_versions.values.uniq.count > 1 %>">
```

Comment:
No, nothing went wrong, it's just a visual cue to the user that there are multiple versions of a project deployed to this environment. It's also a valid state for a slow-rollout deployment strategy.